res/database/Default/lang/de.xml

```diff
@@ -5,14 +5,78 @@
 		<person guid="common-amateur-actors" first_name="" last_name="Laiendarsteller" nick_name="Laiendarsteller" />
 		<person guid="common-amateur-director" first_name="" last_name="Regiepraktikant" nick_name="Regiepraktikant" />
 		<person guid="unnamed-person" first_name="" last_name="Unbekannt" nick_name="Unbekannt" />
+		<person guid="various-voice-artists" first_name="" last_name="versch. Stimmkünstler" />
+		<person guid="various-puppeteers" first_name="" last_name="versch. Puppenspieler" />
+		<person guid="various-models" first_name="" last_name="versch. Models" />
+		<person guid="various-musicians" first_name="" last_name="versch. Musiker" />
 		<person guid="1994aa3a-23c3-48ce-b20a-d1f021df8c63" first_name="Königin Svetlana" last_name="von Schweden" />
 		<person guid="2994ff3d-ffc4-f8ce-c25a-e1g0d1df8c64" first_name="ein Arbeitsloser namens" last_name="„Groggy“" />
 		<person guid="f14d6ba0-0db5-4a76-a4d3-daba1603e4f6" first_name="Archie" last_name="Arbeitslos" />
 		<person guid="f43f2b92-43cd-4fc8-93aa-25adc647fb63" first_name="Erwin" last_name="Rotnase" />
+
+		<person guid="51397467-86e2-4c5f-9226-d93ecf9288a6" first_name="Nikola Sergejewitsch" last_name="Crasstschew" />
+		<person guid="6dd0a9b9-96a0-4712-83f0-f6388bdc5bec" first_name="Mihail" last_name="Gorbotschew" />
 	</persons>
 	<roles>
 		<!--
 			<role guid="script-roles-ron-001" first_name="Vincent" last_name="Graf" title="" />
 		-->
+
+		<role guid="a00ea58a-4a9f-466e-9fe5-cf8b53730b60" first_name="Schneewittchen" last_name="" />
+		<role guid="e38dc13c-ce6a-4a0f-98f5-fd5cec332bf6" first_name="Rosenrot" last_name="" />
+		<role guid="adee8709-5aa4-465b-af8d-60d4ee3e8847" first_name="Märchenprinz" last_name="" />
+		<role guid="5cda9a5e-f823-42b9-993b-28d3011d76e5" first_name="Blaubart" last_name="" />
+		<role guid="847e75c4-c92a-4539-a606-4adbf3c84e44" first_name="Goldlöckchen" last_name="" />
+		<role guid="b3ec26ce-9105-4545-b66f-c08dc5ce91bb" first_name="Rotkäppchen" last_name="" />
+		<role guid="53261965-0118-4585-86a0-285d5f8bfc18" first_name="Aschenputtel" last_name="" />
+		<role guid="dfc4b44e-471f-4b4d-83a0-a91cf0ca8dec" first_name="Sindbad" last_name="der Seefahrer" />
+		
+		<role guid="9e026c4a-04e2-41e9-8865-a8753d53ef01" first_name="Nordwind" last_name="" />
+
+		<role guid="roles-popeye-olive" first_name="Liv Oilia" last_name="" />
```

Comment:
different split of first/last name than in English intended?

Reply:
yes intentional, character seems to have a last name in original English version but not in German

Reply:
This meanst that referencing the name via the role must be done very carefully - referencing first name or full name is fine, but referencing the last name only will cause problems.

Reply:
Not much of a problem here in practice, that character likely will never be referenced by last name only.